Assets/MixedRealityToolkit.SDK/Experimental/Dialog/Scripts/Dialog.cs

```diff
@@ -0,0 +1,232 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.Dialog
+{
+    public abstract class Dialog : MonoBehaviour
+    {
+        protected DialogResult result;
+
+        protected DialogState state = DialogState.Uninitialized;
+        public DialogState State
+        {
+            get
+            {
+                return state;
+            }
+            set
+            {
+                state = value;
+            }
+        }
```

Comment:
minor change, can you simplify usage to following format to save space/readability and also add a comment
get => state;
set => state = value;

Thanks!